policies/unsized-media.md

```diff
@@ -2,7 +2,7 @@
 
 loonybear@, last updated: 4/25/2018
 
-<span style="color:#38761d;">Status: Draft Proposal</span>
+<span style="color:#38761d;">Status: feature landed behind a flag</span>
```

Comment:
Can you change this to say that it is available "in Chrome" behind a flag? -- The feature itself is still just a proposal from a standards perspective, even if it has an experimental implementation in one browser right now.